gr-filter/lib/fft_filter.cc

```diff
@@ -24,6 +24,7 @@
 #include "config.h"
 #endif
 
+#include <boost/smart_ptr/make_unique.hpp>
```

Comment:
Meh. I wonder if we should go straight to C++14, or create our own `std::make_unique` for C++11 as a stop-gap.